### PR TITLE
Issue/#10 move to src

### PR DESCRIPTION
**Pull request by camoore1979**

### CODE REVIEW

**Comment by camoore1979** [comment]
This was fun... back to some coding.

Basic idea was to:

1. abstract logic to separate functions in separate modules
2. implement async / await instead of callbacks
3. added plain / basic eslint instead of eslint-config-polaris (since this is an open source tool)

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    console.error(err);
+    process.exit(1);
+  }
+}
+
+main();
+
+process.on('exit', (code) => {
+  if (code === 0) {
+    console.log('processed executed successfully.')
```
Comment (jeffbum) [inline src/main.js]: console.log('process executed successfully.')
```suggestion
    console.log('processed executed successfully.')
```

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  } else {
+    console.log('processed exited unsuccessfully...')
```
Comment (jeffbum) [inline src/main.js]: console.log('process exited unsuccessfully...')
```suggestion
    console.log('processed exited unsuccessfully...')
```

File: src/queries/getTableStructure.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+         cols.udt_name, 
+         cols.character_maximum_length, 
+         cols.datetime_precision,
+         (select MAX(pg_catalog.col_description(oid,cols.ordinal_position::int)) from pg_catalog.pg_class c where c.relname=cols.table_name) as column_comment
+  from information_schema.columns cols
+  where cols.table_catalog='${dbName}' and cols.table_name='${tableName}'`;
+
+module.exports = async function getTableStructure(tableName) {
+  const query = tableColumnInfoQuery(db.dbName, tableName);
+  // console.log(`query: ${query}`)
```
Comment (jeffbum) [inline src/queries/getTableStructure.js]: can remove console log.

File: src/funcs/getDbStructure.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    const getAllTableColumns = tables.map(async tableName => {
+      const structure = await getTableStructure(tableName);
+      return {
+        tableName,
+        structure
+      };
+    });
+
+    console.log(`getting structure of schema "${schema}"`);
+    const allstructures = await Promise.all(getAllTableColumns);
```
Comment (jeffbum) [inline src/funcs/getDbStructure.js]: camel case

File: src/funcs/transformToDbml.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+const { EOL } = require('os');
+
+// TODO: possible to use https://www.dbml.org/js-module/#api for the transform to dbml?
```
Comment (jeffbum) [inline src/funcs/transformToDbml.js]: They use sql statements such as `create table tableName fields, fields, fields, etc.`, and not json.

**Review by jeffbum — APPROVED** [review]
just a few wording suggestions. Also, some commented out code could be removed (the console logs, etc.). Also, do we want to convert the todos into issues? That way it keeps the base clean, and we still have visibility? Nice work man.

**Pull request merged**